Backend as a Service Stubs (#3101)
withastro/docs: src/content/docs/en/guides/backend.mdx

---
title: Use a backend service with Astro
description: How to use a backend service to add authentication, storage and data
i18nReady: true
---
import BackendGuidesNav from '~/components/BackendGuidesNav.astro';

> **Ready to add a backend to your Astro project?** Follow one of our guides to integrate a backend service.

Reviewer: ```suggestion
**Ready to add auth, storage or data to your Astro project?** Follow one of our guides.
```
